tests/run-multitests.py

```diff
@@ -394,6 +403,8 @@ def main():
     for i in cmd_args.instance:
         if i.startswith("exec:"):
             instances_test.append(PyInstanceSubProcess([i[len("exec:") :]]))
+        elif i == "unix":
+            instances_test.append(PyInstanceSubProcess([MICROPYTHON]))
```

Comment:
This will work also on windows (and other systems), so maybe it should be called something else... but something like "local_micropython" is long.  Could also add "cpython" because it can be useful to run against CPython sometimes.

Reply:
how about "micropython" and "cpython" ? (which matches the names of the constants anyway)

Reply:
Yeah, that makes the most sense.  Could also have "micropython-coverage" then.